### PR TITLE
Added open until and closed until to resource list

### PR DESCRIPTION
**Pull request by twolfe2**

### CODE REVIEW

**Comment by twolfe2** [comment]
Fixes #141

File: app/components/Search/ResourcesRow.jsx [hunk 1]
```diff
@@ -35,6 +35,55 @@ class ResourcesRow extends Component {
...
+      closingOpeningTimes[item.day.replace(/,/g, '')] = {
+        open: item.opens_at,
+        close: item.closes_at
+      };
+    });
+
+    if(closingOpeningTimes[currDay] && hour < closingOpeningTimes[currDay].close) {
+      return {
+        open:true, 
+        time: timeToString(closingOpeningTimes[currDay].close)
```
Comment (ZanderSparrow) [inline app/components/Search/ResourcesRow.jsx]: this is the closing time, right? maybe change to something more descriptive like 'closingAt'.

File: app/components/Search/ResourcesRow.jsx [hunk 2]
```diff
@@ -35,6 +35,55 @@ class ResourcesRow extends Component {
...
+            open: false,
+            time: `${timeToString(closingOpeningTimes[d].open)} ${d}`
+          };
+          return true;
+        }
+      });
+
+      if(result.time) return result;
+
+      days.some(d => {
```
Comment (ZanderSparrow) [inline app/components/Search/ResourcesRow.jsx]: not sure how this is different from the block above (other than splicing out today). If the resource is closed (so return not yet reached) why not just jump to here and skip the else above?

File: app/components/Search/ResourcesRow.jsx (hunk 1 above)
Comment (twolfe2) [inline app/components/Search/ResourcesRow.jsx]: Good idea. I'll apply this change

File: app/components/Search/ResourcesRow.jsx (hunk 2 above)
Comment (twolfe2) [inline app/components/Search/ResourcesRow.jsx]: The first loop checks to see if the resource is open any days that are remaining in the week. 

If no open days are found for the remainder of the week, I start at the beginning of the week and check to see the soonest day the resource is open.

Comment (ZanderSparrow) [inline app/components/Search/ResourcesRow.jsx]: I see--You want the next open day, so you want to start with the days after the current day. You could do it in one loop by splitting the array at the current day and then putting them back together with current day at the end. This is also fine, though.

**Pull request merged**